### DISCUSSION TITLE
Question: Is there a way to cycle through #tabs with spaces?

### DISCUSSION THREAD

**Discussion by shmup** [scandum/tintin#110] [Script help]
Example: `#tab {talker log on}; #tab {talker log off};`

In this video I'm just typing `talk` then tabbing a bunch.

https://user-images.githubusercontent.com/118710/183765824-e3f7c961-c2b8-4e64-9240-13d3027573f7.mp4

Kind of a silly question, I know there are other ways to do this. Even if just using `talker-log-on` and `talker-log-off`.

I just know I've become a big fan of adding a `#tab` for all my aliases, and am leaning on it a bunch.

**Comment by scandum** [comment]
As far as I know it can't be done.

**Reply by shmup** [reply]
Thanks for all your feedback with me lately @scandum, I get back into a mud then become obsessed with thinking about tintin.. more than playing the mud, lol
